physutils/tasks.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Helper class for holding physiological data and associated metadata information."""
+
+import logging
+
+from .io import load_from_bids, load_physio
+from .physio import Physio
+from .utils import is_bids_directory
+
+# from loguru import logger
+
+try:
+    from pydra import task
```

Comment:
@smoia Is it possible that here it should be from `pydra.mark import task` instead of `from pydra import task` ?

Reply:
With that change `test_generate_physio_bids_file` and `test_generate_physio_auto` in `test_tasks.py` are now passing :thinking:

Reply:
Yes that could be it actually. Thank you for catching it!